modules/gapi/include/opencv2/gapi/streaming/onevpl/utils.hpp

```diff
@@ -0,0 +1,37 @@
+// This file is part of OpenCV project.
```

Comment:
Strongly disagree  about adding this kind of utils.hpp/cpp

Reply:
Maybe we should call it "simple" or "default" .hpp? :)

Reply:
Done